loader/loader.c

```diff
@@ -5284,7 +5427,19 @@ VkResult loader_create_instance_chain(const VkInstanceCreateInfo *pCreateInfo, c
 
         create_info_disp.pNext = loader_create_info.pNext;
         loader_create_info.pNext = &create_info_disp;
-        res = fpCreateInstance(&loader_create_info, pAllocator, created_instance);
+
+	VkLayerInstanceCreateInfo create_info_disp2;
+
+        create_info_disp2.sType = VK_STRUCTURE_TYPE_LOADER_INSTANCE_CREATE_INFO;
+        create_info_disp2.function = VK_LOADER_LAYER_CREATE_DEVICE_CALLBACK;
+
+        create_info_disp2.u.layerDevice.pfnLayerCreateDevice = vkLayerCreateDevice;
+        create_info_disp2.u.layerDevice.pfnLayerDestroyDevice = vkLayerDestroyDevice;
```

Comment:
This section shows compiler errors on our Windows tests. That's just because your headers pull request hasn't been merged. It does mean that after that happens we'll probably want to rebase this PR so that we can verify the tests pass.